### ISSUE TITLE
Keychain

### ISSUE THREAD

**Issue by SuperMario64fan** [crumblingstatue/FloweysTimeMachine#34]
There is no way to add things to the keychain like the Asgore keys or Sans' keys.

**Comment by jD91mZM2** [comment]
In the core, the keychain is actually just a bunch of flags saying "has asgore's key?"  
It'd take some effort to add all of them.

**Comment by ackledotdev** [comment]
Think this was completed too.

**Comment by gfrhj** [comment]
The ultimate necropost LOL

**Comment by KockaAdmiralac** [comment]
No, it was not completed. Please stop leaving unhelpful replies to old issues.
